packages/recommend-core/src/getRecommendations.ts

```diff
@@ -47,7 +47,7 @@ function getDefaultedProps<TObject>(
   props: GetRecommendationsProps<TObject>
 ): GetRecommendationsInternalProps<TObject> {
   return {
-    fallbackFilters: [],
+    fallbackParameters: { facetFilters: [] },
```

Comment:
We either need to deeply default it (because if user provides `facetFilters: {}`, the current flow would throw), or handle code path expecting it to also be `undefined`. I'm fine with either way since this code will be thrown next week.

```suggestion
    fallbackParameters: {},
```

Reply:
There was some small issues when defaulting it to an empty object, does the other way around works to you? https://github.com/algolia/ui-components/pull/38/commits/496799dbe425f9eafbbf513703d6c369a9165d97

I thought it would be easier than adding conditions